app/code/community/Bolt/Boltpay/Model/Payment.php

```diff
@@ -633,6 +514,190 @@ public function handleTransactionUpdate(Mage_Payment_Model_Info $payment, $newTr
             throw $e;
         }
     }
+    
+    public function handleRefundTransactionUpdate(
+        Mage_Payment_Model_Info $payment,
+        $newTransactionStatus,
+        $prevTransactionStatus,
+        $transactionAmount = null,
+        $transaction
+    ) {
+        try {
```

Comment:
There's a bit of duplication here at the beginning that might be able to be refactored out if it was handled in `handleTransactionUpdate`